src/azul/indexer/action_controller.py

```diff
@@ -54,9 +55,15 @@ def _handle_events(self,
                        event: Iterable[SQSRecord],
                        message_handler: Callable[[A, JSON], None]):
         for record in event:
-            message = SQSMessage.from_record(record)
-            log.info('Worker handling message %r, attempt #%i (approx), message ID %s',
-                     message.body, message.attempts, message.id)
+            message: SQSMessage | SQSFifoMessage
```

Comment:
What's this for?  `Fruit | Apple` is equivalent to just `Fruit`.

Reply:
Agreed at PL to change annotation to `SQSMessage` and live with the false-positive PyCharm warning.